rotatephi.cpp

```diff
@@ -33,7 +33,7 @@ void parse_args_and_init(int argc, char** argv);
 // TODO: add verbosity argument
 int main (int argc, char** argv) {
   int verbose = 0;
-  
+  printf("M_PI=%f",M_PI);
```

Comment:
I'm guessing this was here for debugging purposes, correct? Could you please remove this line?
When you then push to your branch, the pull-request will be updated automatically.